internal/business/server/openapi.go

```diff
@@ -152,14 +168,11 @@ func (s *openAPIServer) Redirect(ctx context.Context, req openapi.RedirectReques
 	}, nil
 }
 
-func (s *openAPIServer) toErrorModel(err error) (model openapi.ErrorModel, httpStatus int) {
+func (s *openAPIServer) toErrorModel(err error) (string, int) {
 	var serviceErr *serviceerr.Error
 	if !errors.As(err, &serviceErr) {
 		serviceErr = serviceerr.ErrUnknown
 	}
 
-	return openapi.ErrorModel{
-		ErrorCode: (*int)(&serviceErr.Code),
-		ErrorMsg:  &serviceErr.Message,
-	}, serviceErr.HTTPStatus()
+	return serviceErr.Message, int(serviceErr.Code)
```

Comment:
Why was it changed? The method was intended to return an HTTP status to map the internal error code to the HTTP response code. Now the method returns incorrect results and therefore the service returns incorrect HTTP codes